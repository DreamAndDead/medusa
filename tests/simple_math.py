print(5 + 3)
print(18 - 2)
print(5 * 5)
print(64 / 2)
print(11 ** 2)
print(11 // 2)
print(11 / 2)
print(((5 + 34) ** 2 / 53) * (24 - 6 * 3))
