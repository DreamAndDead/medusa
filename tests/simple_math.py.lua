dofile('./pythonlua/luainit.lua')

print((5 + 3))
print((18 - 2))
print((5 * 5))
print((64 / 2))
print((math.pow(11, 2)))
print((math.floor(11 / 2)))
print((11 / 2))
print((((math.pow((5 + 34), 2)) / 53) * (24 - (6 * 3))))