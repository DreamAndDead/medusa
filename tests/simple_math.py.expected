8
16
25
32
121
5
5.5
172.18867924528
